Sources/SwiftCurrent_SwiftUI/Views/ViewControllerWrapper.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ViewControllerWrapper.swift
+//  
```

Comment:
Missing the module
```suggestion
//  SwiftCurrent_SwiftUI
//
```

Reply:
If this is going to be a requirement that trips up code review it should be added to our linter.